etna/core/static/admin/js/inputLengthIndicators.js

```diff
@@ -0,0 +1,69 @@
+const lengthIndicatorClassname = "inputlengthindicator";
+const lengthIndicatorExceededClassname = "inputlengthindicator--exceeded";
+const countHTMLBase = '<span class="w-sr-only">Character count:</span> ';
+
+const countChars = function (text) {
+  /*
+  Count characters in a string, with special processing to account for astral symbols in UCS-2. See:
+  - https://github.com/RadLikeWhoa/Countable/blob/master/Countable.js#L29
+  - https://mathiasbynens.be/notes/javascript-unicode
+  - https://github.com/tc39/proposal-intl-segmenter
+  */
+    if (text) {
+        // Find as many matches as there are (g), matching newlines as characters (s), as unicode code points (u).
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#advanced_searching_with_flags.
+        const matches = text.match(/./gsu);
+        return matches ? matches.length : 0;
+    }
+    return 0;
+};
+
+const updateLengthIndicator = function (
+    lengthIndicator,
+    length,
+    maxChars,
+) {
+    lengthIndicator.innerHTML = countHTMLBase + `${length}/${maxChars}`;
+
+    if (length > maxChars) {
+        lengthIndicator.classList.add(lengthIndicatorExceededClassname);
+    } else {
+        lengthIndicator.classList.remove(lengthIndicatorExceededClassname);
+    }
+};
+
+const initializeLengthIndicator = function (input) {
+    // Look for relevant attributes on the input
+    const maxChars = input.getAttribute("maxlength");
+
+    // Add hidden HTML element to display the count
+    const lengthIndicator = document.createElement("div");
+    lengthIndicator.className = lengthIndicatorClassname;
```

Comment:
Suggestion: `classList.add` allows for multiple classes so you can shorten this to `classList.add(lengthIndicatorClassname, "w-help-text")`

Reply:
Something new learned :) Implemented in [be92ad4](https://github.com/nationalarchives/ds-wagtail/pull/736/commits/be92ad4b57dacab6b70a2fe62d537bbdca8069a3). Cheers Will!